python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -880,6 +883,43 @@ keys:
                       max: 65535
                       convert_types:
                         - str
+            aaa:
+              type: dict
+              keys:
+                unresponsive:
+                  type: dict
+                  description: Configure AAA timeout options.
+                  keys:
+                    eap_response:
+                      type: str
+                      description: EAP response to send. EOS default is `success`.
```

Comment:
```suggestion
                      description: EAP response to send.
                      default: "success"
```

Reply:
Generally we dont add default in schema for eos defaults.

Reply:
yeah, but here I think its better to add in default than description and the extension also takes default from schema.
Let's confirm with @ClausHolbechArista and @guillaumeVilar once.

Reply:
Laxmikant is correct here, we should not implement default values in the cli_gen schema.